### PR TITLE
Matrix term label hover over

### PR DESCRIPTION
**Pull request by congyu-lu**
## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [x] Tests: added and/or passed unit and integration tests, or N/A
- [x] Todos: commented or documented, or N/A
- [x] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

File: client/plots/matrix.js [hunk 1]
```diff
@@ -659,6 +659,16 @@ export class Matrix {
 		for (const t of this.termOrder) {
 			const countedSamples = new Set()
 			t.counts = { samples: 0, hits: 0 }
+			if (t.tw.term.type == 'geneVariant' || t.grp.name == 'Cytogenetics') {
```
Comment (xzhou82) [inline client/plots/matrix.js:662]: shouldn't hardcode grp name

Comment (congyu-lu) [reply client/plots/matrix.js]: Removed 'Cytogenetics'

**Comment by xzhou82** [comment]
over row label, hover and click should not share the same menu, otherwise you will never be able to edit. create a new one for hover

**Comment by congyu-lu** [comment]
> over row label, hover and click should not share the same menu, otherwise you will never be able to edit. create a new one for hover

changed to hovering over won't dismiss edit menu

**Review by xzhou82 — APPROVED** [review]
thanks it works nicely. i would like you to add comments to explain the logic and purpose, where it's not obvious

**Pull request merged**